include/swift/Sema/CSFix.h

```diff
@@ -2061,6 +2064,30 @@ class MustBeCopyable final : public ConstraintFix {
   }
 };
 
+class AllowInvalidPackElement final : public ConstraintFix {
+  Type packElementType;
+
+  AllowInvalidPackElement(ConstraintSystem &cs, Type packElementType,
+                          ConstraintLocator *locator)
+      : ConstraintFix(cs, FixKind::AllowInvalidPackElement, locator),
+        packElementType(packElementType) {}
+
+public:
+  std::string getName() const override {
+    return "allow concrete pack element";
+  }
+
+  bool diagnose(const Solution &solution, bool asNote = false) const override;
+
+  static AllowInvalidPackElement *create(ConstraintSystem &cs,
+                                         Type packElementType,
+                                         ConstraintLocator *locator);
+
+  static bool classof(ConstraintFix const* fix) {
```

Comment:
Nit: all other fixes are using `const ConstraintFix *`.

Reply:
Hah, I copied this from the fix right above this one

```cpp
  static bool classof(ConstraintFix const* fix) {
    return fix->getKind() == FixKind::MustBeCopyable;
  }
```

Reply:
Could you please adjust that one as well then? :)

Reply:
Yep, already working on a commit!

Reply:
https://github.com/apple/swift/pull/64122